tracee-ebpf/main.go

```diff
@@ -1037,8 +1069,8 @@ func locateFile(file string, dirs []string) string {
 	return ""
 }
 
-// getBPFObject finds or builds ebpf object file and returns it's path
-func getBPFObject() (string, error) {
+// getBPFObjectPath finds or builds ebpf object file and returns it's path
+func getBPFObjectPath() (string, error) {
```

Comment:
As discussed, we want to have to TRACEE_BPF_FILE to be looked also in core mode. I think that we can call this function an exit before the binary build (or just split it into two functions)